### ISSUE TITLE
[DomTools] Bring back I3IsolatedHitsCutModule (Trac #1397)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1389]
Or at least provide an alternative for filter scripts IceCube_BaseProc.py.

Migrated from https://code.icecube.wisc.edu/ticket/1397
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:07", 
    "description": "Or at least provide an alternative for filter scripts IceCube_BaseProc.py.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335647931556", 
    "component": "combo core", 
    "summary": "[DomTools] Bring back I3IsolatedHitsCutModule", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-10-14T07:06:54", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151028T20:32:04: 
* olivas commented:

> In changeset 138911:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138911"
> brought back I3IsolatedHitsCutModule.  closes #1389.
> ```


* olivas changed status from "new" to "closed"
* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:07: 
* olivas commented:

> Milestone November 2015 deleted


* olivas changed milestone from "November 2015" to ""
